src/app/containers/Text/index.test.jsx

```diff
@@ -35,6 +36,11 @@ describe('TextContainer', () => {
       ],
     };
 
-    shouldMatchSnapshot('should render correctly', <TextContainer {...data} />);
+    shouldMatchSnapshot(
+      'should render correctly',
+      <StaticRouter>
```

Comment:
An explanatory comment about why `StaticRouter` is needed here would be useful, for devs who maintain these tests in future.